tests/integration/nm/linux_bridge_test.py

```diff
@@ -52,42 +95,21 @@ def bridge_default_config():
                     LB.STP_PRIORITY: 32768,
                 },
             },
-            LB.PORT_SUBTREE: [],
+            LB.PORT_SUBTREE: ports_states,
         }
     }
 
 
-def test_create_and_remove_minimum_config_bridge(bridge_minimum_config):
-    bridge_desired_state = bridge_minimum_config
-
-    with _bridge_interface(bridge_desired_state):
-
-        bridge_current_state = _get_bridge_current_state()
-        assert bridge_current_state
-        assert not bridge_current_state[LB.CONFIG_SUBTREE][LB.PORT_SUBTREE]
-
-    assert not _get_bridge_current_state()
-
-
-def test_create_and_remove_bridge(port0_up, bridge_default_config):
-    bridge_desired_state = bridge_default_config
-
-    port_name = port0_up[schema.Interface.KEY][0][schema.Interface.NAME]
-    port = {
-        LB.PORT_NAME: port_name,
-        LB.PORT_STP_PRIORITY: 32,
-        LB.PORT_STP_HAIRPIN_MODE: False,
-        LB.PORT_STP_PATH_COST: 100,
-    }
-
-    bridge_desired_state[LB.CONFIG_SUBTREE][LB.PORT_SUBTREE].append(port)
-
-    with _bridge_interface(bridge_desired_state):
-
-        bridge_current_state = _get_bridge_current_state()
-        assert bridge_desired_state == bridge_current_state
-
-    assert not _get_bridge_current_state()
+def _create_bridge_ports_config(ports):
+    return [
+        {
+            LB.PORT_NAME: port,
+            LB.PORT_STP_PRIORITY: 32,
```

Comment:
wouldn't we want to parameterize  these other values ? (just asking, got no need for it).

Reply:
Not sure what you are asking.
At the moment this is not changing the logic, just refactoring the code without changes to behaviour.

If we will need to change the ports properties, we will probably need to add options to do so in the future.